custom_components/ha_kia_hyundai/climate.py

```diff
@@ -90,6 +90,10 @@ async def async_set_hvac_mode(self, hvac_mode: HVACMode) -> None:
                     set_temp=int(self.target_temperature),
                     defrost=self.coordinator.climate_desired_defrost,
                     heating=self.coordinator.climate_desired_heating_acc,
+                    driver_seat=self.coordinator.desired_driver_seat_comfort,
```

Comment:
same thing here

Reply:
So this conversion to SeatSettings is now done as part of the select option. It will either have a SeatSetting or will be None otherwise. I think that should work, let me know if not.

```
 async def async_select_option(self, option: str) -> None:
        """Change the select option."""
        setattr(
            self.coordinator,
            self.entity_description.key,
            SeatSettings(STR_TO_NUMBER[option]),
        )
        self._attr_current_option = option
        self.async_write_ha_state()
```